public/svgs/frost/infobar-create.svg

```diff
@@ -1,6 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we should just have 'plus' and 'find' icons and just scale them to the scenarios that require those.  Please raise an issue to work on this.